indico/modules/rb_new/client/js/modules/calendar/selectors.js

```diff
@@ -29,3 +29,4 @@ export const getCalendarData = createSelector(
         rows: rows.map(entry => ({...entry, room: allRooms[entry.room_id]}))
     })
 );
+export const getFilters = ({['calendar']: stateSlice}) => stateSlice.filters;
```

Comment:
`{calendar: stateSlice}` - the `[...]` is only needed if you want to use an expression there

Reply:
actually, even simpler:

    export const getFilters = ({calendar}) => calendar.filters;